tests/domains_email_forwards_test.rs

```diff
@@ -19,18 +19,19 @@ fn test_list_email_forwards() {
         .unwrap();
     let email_forwards_list = response.data.unwrap();
 
-    assert_eq!(2, email_forwards_list.len());
+    assert_eq!(1, email_forwards_list.len());
```

Comment:
Usually, the fixtures are updated to match the specs as they are applied across repos instead of updating the specs.

Reply:
Fair point. Let's remember this for next time, as all clients have been already updated with this change already 👍